nss-day-cohort-45/nutyshell-running-with-scissors-1
# Rcd style articles & dialog boxes

# Description
-Styling added to article cards 
-Styling added to dialog boxes
-Most buttons switched out with icons
-IMPORTANT: (main.css) set height to 445px for all lists. Check to see if this is appropriate. If not, can push up a new commit to revert it back to height: 100%

## Type of change

# Testing Instructions
-Run JSON server
-See new icon buttons, woo
-Add a new article to see if dialog box is styles along with having a new cancel icon
-Edit an article to see if dialog box is styled along with having a new cancel icon
-Check to see if the articles are listed starting at the top with the most recent.
-Article cards should now have a slight shadow behind them
-See if the static height of all of the list (less the friend's list) is appropriate. (This is on line 50 in main.css)

# Checklist:
